src/core/tuning/commands.ts

```diff
@@ -44,7 +46,7 @@ export async function cloneWithNewName(srcUri: vscode.Uri) {
 
   const tuningFsPath = path.join(
     path.dirname(srcUri.fsPath),
-    `${newFilename.replace(/^[^:]*:/, "")}.xml`
+    sanitize(newFilename + ".xml", { replacement: '_', })
```

Comment:
Sanitizing is a great idea and I will include it, but the purpose of the `replace(/^[^:]*:/, "")` was *not* sanitization, but rather removing the creator name prefix (since many people use the naming convention `CreatorName_ModName:BlahBlahBlah`).

I know the regex isn't the most elegant solution, but both are needed here. Maybe a helper function that does both?

Reply:
Why is the creator name prefix removed here but not in Folder to S4TK Project?